UPGRADE-0.11.md

```diff
@@ -16,3 +16,21 @@ UPGRADE FROM 0.10 to 0.11
    - Installing OverblogGraphiQLBundle
      - `composer require --dev overblog/graphiql-bundle`
      - Follow instructions at https://github.com/overblog/GraphiQLBundle
+   - In case you have defined the `versions` in your configuration
+     - Remove it from `overblog_graphql`
+         ```diff
+         overblog_graphql:
+         -    versions:
+         -        graphiql: "0.11"
+         -        react: "15.6"
+         -        fetch: "2.0"
+                  relay: "classic"
```

Comment:
relay entry can also be deleted, i will add an option directly on dump command

Reply:
Wanna make it after or before this pr?

Reply:
This can be done after I think.